assets/scss/custom/_type.scss

```diff
@@ -221,6 +221,15 @@ dl {
     color: $text-color;
 }
 
-
-
+.template-product .panel label,
+.product-pull-left,
+.property-label {
+    font-family: Helvetica Neue;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 13px;
+    line-height: 20px;
+    color: #757575;
```

Comment:
can we use variable here? we have $text-muted variable for this color
![image](https://user-images.githubusercontent.com/22875828/93853419-20a1aa00-fcb4-11ea-98f0-2d5fe7f07c19.png)